config/ext_jenkins-infra-jobs.yaml

```diff
@@ -121,6 +121,25 @@ jobsDefinition:
             fileName: "kubeconfig"
             description: "Kubeconfig file for temp-privatek8s"
             secretBytes: "${base64:${KUBECONFIG_TEMP_PRIVATEK8S}}"
+      kubernetes-management-updatecli:
+        name: Kubernetes Management Dependencies
+        repository: kubernetes-management
```

Comment:
@dduportal At first I've wondered how I could specify a second job on the same repository, looked at your jenkins-jobs chart and found how the id or the .repository value of the job are coalesced. Again, really nice job! 👏 (pun intended)